### PR TITLE
Sample app: multi lists activity added, clips added

### PR DESCRIPTION
**Pull request by fnabrdal**

### CODE REVIEW

File: app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -19,15 +19,34 @@
 
                 <category android:name="android.intent.category.LAUNCHER" />
             </intent-filter>
+            <intent-filter android:label="Storyteller Links" android:autoVerify="true">
```
Comment (DaveMiscampbell) [inline app/src/main/AndroidManifest.xml:22]: We shouldn't have this label

File: app/src/main/AndroidManifest.xml [hunk 2]
```diff
@@ -19,15 +19,34 @@
...
+                <action android:name="android.intent.action.VIEW" />
+                <category android:name="android.intent.category.DEFAULT" />
+                <category android:name="android.intent.category.BROWSABLE" />
+
+                <data
+                    android:scheme="https"
+                    android:host="stormideasqa.ope.nstori.es"
```
Comment (DaveMiscampbell) [inline app/src/main/AndroidManifest.xml:29]: Let's make this `storyteller.ope.nstori.es` (and throughout)

File: app/src/main/java/com/example/storytellerSampleAndroid/MainActivity.kt [hunk 3]
```diff
@@ -143,10 +121,10 @@ class MainActivity : AppCompatActivity(R.layout.activity_main), StorytellerDeleg
             "Storyteller Sample",
             "onDataLoadComplete callback: success $success, error $error, dataCount $dataCount"
         )
-        refreshLayout.isRefreshing = false
-        if (success) {
-            handleDeepLink(intent?.data)
-        }
+        binding.refreshLayout.isRefreshing = false
+//        if (success) {
```
Comment (DaveMiscampbell) [inline app/src/main/java/com/example/storytellerSampleAndroid/MainActivity.kt:125]: Probably just remove this bit?

Comment (fnabrdal) [reply app/src/main/java/com/example/storytellerSampleAndroid/MainActivity.kt:125]: yes

File: app/src/main/AndroidManifest.xml [hunk 4]
```diff
@@ -19,15 +19,34 @@
 
                 <category android:name="android.intent.category.LAUNCHER" />
             </intent-filter>
-            <intent-filter android:label="Storyteller Links">
+            <intent-filter android:autoVerify="true">
                 <action android:name="android.intent.action.VIEW" />
                 <category android:name="android.intent.category.DEFAULT" />
                 <category android:name="android.intent.category.BROWSABLE" />
-            <!-- adb shell am start -W -a android.intent.action.VIEW -d "https://storytellersampleapp/[PAGE_ID]" -->
```
Comment (abd3lraouf) [inline app/src/main/AndroidManifest.xml:26]: maybe read this comment and add more information about deeplinking

**Review by abd3lraouf — APPROVED** [review]

File: app/src/main/AndroidManifest.xml (hunk 4 above)
Comment (fnabrdal) [reply app/src/main/AndroidManifest.xml:26]: done

Comment (abd3lraouf) [reply app/src/main/AndroidManifest.xml:26]: Thank you so much

**Pull request merged**